jwala-services/src/main/java/com/cerner/jwala/control/jvm/command/JvmCommandFactory.java

```diff
@@ -198,10 +198,13 @@ private ExecCommand getExecCommandForHeapDump(String scriptName, Jvm jvm) {
      * @return
      */
     private ExecCommand getExecCommandForThreadDump(String scriptName, Jvm jvm) {
+        final String trimmedJvmName = StringUtils.deleteWhitespace(jvm.getJvmName());
         final String jvmRootDir = Paths.get(jvm.getTomcatMedia().getRemoteDir().toString() + '/' +
                 StringUtils.deleteWhitespace(jvm.getJvmName()) + '/' + jvm.getTomcatMedia().getRootDir())
                 .normalize().toString();
-        return new ExecCommand(getFullPathScript(jvm, scriptName), jvm.getJavaHome(), jvmRootDir, jvm.getJvmName());
+        final DateTimeFormatter formatter = DateTimeFormat.forPattern("yyyyMMdd.HHmmss");
+        final String dumpFile = "threadDump." + trimmedJvmName + "." + formatter.print(DateTime.now());
```

Comment:
I missed this before. Can you rename the generated file and add an extension .txt to it, since most of our users are windows users?
Rename
thread_dump_01-24-2018-12.21.23
to something like
thread-dump-01-24-2018-12-21-23.txt